rest-api/dao/participant_dao.py

```diff
@@ -344,6 +344,34 @@ def add_missing_hpo_from_site(self, session, participant_id, site_id):
     # Update the version and add history row
     self._do_update(session, participant, participant)
 
+  def switch_to_test_account(self, session, participant_id):
+    test_hpo = HPODao().get_by_name(TEST_HPO_NAME)
+    if test_hpo is None:
+      raise BadRequest('No test hpo found for name ' + TEST_HPO_NAME)
+
+    test_hpo_id = test_hpo.hpoId
+    site = self.site_dao.get_by_hpo_id(test_hpo_id)
+    if site is None:
+      raise BadRequest('Invalid siteId reference hpo_id %r.' % test_hpo_id)
+
+    participant = self.get_for_update(session, participant_id)
+    if participant is None:
+      raise BadRequest('No participant %r for HPO ID udpate.' % participant_id)
```

Comment:
nit: %r is fine here but it will default back to the str() representation in this case.